site-src/geps/gep-713.md

```diff
@@ -331,6 +331,53 @@ Although these concerns are not unsolvable, they lead to the conclusion that
 a Kubectl plugin should be our primary approach to providing visibility here,
 with a possibility of adding policies to status at a later point.
 
+### Conditions
+Controllers using the Gateway API policy attachment model SHOULD populate the 
+following condition and reasons to provide a consistent experience across
```

Comment:
nit, tiny clarification on where this should go

```suggestion
following condition and reasons on policy resources to provide a consistent experience across
```

Reply:
thanks, updated!